config.json

```diff
@@ -222,6 +222,11 @@
       "difficulty": 1,
       "slug": "change",
       "topics": []
+    },
+    {
+      "difficulty": 10,
+      "slug": "react",
+      "topics": []
```

Comment:
this track want to add any topics? I suggest a similar list as those in Ceylon https://github.com/exercism/xceylon/blob/master/config.json#L52

Reactive programming, generics, nested classes, higher-order functions?